src/components/ContactsList.vue

```diff
@@ -5,32 +5,84 @@
 
 <template>
 	<AppContentList class="content-list">
+		<NcDialog :open="showDeleteConfirmationDialog"
+			:name="n(
+				'contacts',
+				'Delete {number} contact',
+				'Delete {number} contacts',
+				multiSelectedContacts.size,
+				{ number: multiSelectedContacts.size }
+			)"
+			:buttons="buttons"
+			no-close>
+			{{ t('contacts', 'Are you sure you want to proceed?') }}
+			<span v-if="readOnlyMultiSelectedCount" class="content-list-dialog-warning">
+				{{ n('contacts',
+					'Please note that {number} contact is read only and won\'t be deleted',
+					'Please note that {number} contacts are read only and won\'t be deleted',
+					readOnlyMultiSelectedCount,
+					{ number: readOnlyMultiSelectedCount })
+				}}
+			</span>
+		</NcDialog>
+
 		<div class="contacts-list__header">
 			<div class="search-contacts-field">
 				<input v-model="query" type="text" :placeholder="t('contacts', 'Search contacts …')">
 			</div>
 		</div>
+		<div v-if="isMultiSelecting" class="contacts-list__multiselect-header">
+			<NcButton type="tertiary"
+				:title="n('contacts', 'Unselect {number}', 'Unselect {number}', multiSelectedContacts.size, { number: multiSelectedContacts.size })"
```

Comment:
```suggestion
				:title="n('contacts', 'Unselect {number} card', 'Unselect {number} cards', multiSelectedContacts.size, { number: multiSelectedContacts.size })"
```
Or just
`t('contacts', 'Unselect {number}', { number: multiSelectedContacts.size }`